packages/mui-codemod/src/deprecations/accordion-props/test-cases/expected.js

```diff
@@ -0,0 +1,15 @@
+import Accordion from '@mui/material/Accordion';
+import { Accordion as MyAccordion } from '@mui/material';
+
+<Accordion slots={{
+  transition: CustomTransition
+}} slotProps={{
+  transition: { unmountOnExit: true }
+}} />;
+<MyAccordion slots={{
+  transition: CustomTransition
+}} slotProps={{
+  transition: transitionVars
+}} />;
+<Accordion slots={{ root: 'div' }} slotProps={{ root: { className: 'foo' } }} />;
+<MyAccordion slots={outerSlots} slotProps={outerSlotProps} />;
```

Comment:
This looks like it's not working. The `transition` keys are not getting added.

Reply:
Nice catch, fixed!